src/apps/advanced-search/CqlSearchHeader.tsx

```diff
@@ -30,42 +34,77 @@ const CqlSearchHeader: React.FC<CqlSearchHeaderProps> = ({
     }
   }, [initialCql, setCql]);
 
+  // Local state is needed to track input values as plain strings,
+  // since onLocationChange expects a comma-separated string,
+  // while locationFilter location and sublocation are provided as arrays.
+  const [inputValues, setInputValues] = useState({
+    location: locationFilter?.location?.join(", ") ?? "",
+    sublocation: locationFilter?.sublocation?.join(", ") ?? ""
+  });
+
+  const handleInputChange = (
+    name: "location" | "sublocation",
+    value: string
+  ) => {
+    setInputValues((prevValues) => ({
+      ...prevValues,
+      [name]: value
+    }));
+
+    if (name === "location") {
+      onLocationChange(value);
+    }
+    if (name === "sublocation") {
+      onSublocationChange(value);
+    }
+  };
+
   return (
-    <>
+    <form className="advanced-search-cql-form">
       <h1
         className="text-header-h2 advanced-search__title capitalize-first"
         data-cy={dataCy}
       >
         {t("cqlSearchTitleText")}
       </h1>
-      <textarea
-        className="advanced-search__cql-input focus-styling__input"
+
+      <Textarea
+        id="cql"
+        name="name"
```

Comment:
Please exclude `name` if it is not necessary.

It is not something we use for the other form components like `TextInput`. 

I do not think the current value provides any meaning either.